docs/docsite/rst/dev_guide/developing_modules_general_aci.rst

```diff
@@ -0,0 +1,424 @@
+.. _aci_dev_guide:
+
+****************************
+Developing Cisco ACI modules
+****************************
+This is a brief walk-through of how to create new Cisco ACI modules for Ansible.
+
+For more information about Cisco ACI, look at the :ref:`Cisco ACI user guide <aci_guide>`.
+
+What's covered in this section:
+
+.. contents::
+   :depth: 3
+   :local:
+
+
+.. _aci_dev_guide_intro:
+
+Introduction
+============
+Ansible already ships with a large collection of Cisco ACI modules, however the ACI object model is huge and covering all possible functionality would easily cover more than 1500 individual modules.
```

Comment:
Well, I contemplated using some of the drawings, because they are really helpful, but didn't want to go to the trouble getting the rights to reuse them. But a link to the document could be useful. Fact is that I consider people planning to write such modules to know about this already. Anyone responsible for working with Cisco ACI is already trained and possibly even certified as this is not something small shops will be using.

Reply:
I'll add some links, it doesn't hurt.

Reply:
ah, I was thinking of an inline link, but the "see also" section works too